addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -1087,10 +1063,21 @@ public void run() {
                                 vehicle = queryVehicle();
                             }
                         }
+                    } else {
+                        Thread.sleep(250);
+                    }
+                } catch (IOException | InterruptedException e) {
```

Comment:
Indeed, but if you do not mind I prefer to keep it in for the time being. That is, until I finally get this Tesla back-end stuff stable. I just want to keep track of any errors in the binding whilst it is running in my operational environment. The point is that debugging the Tesla binding without driving is impossible, and I can not drive with the laptop on my lap....